tests/wamr-test-suites/test_wamr.sh

```diff
@@ -328,7 +329,14 @@ function spec_test()
     git checkout -B main
     # [spec] Update note on module initialization trapping (#1493)
     git reset --hard 044d0d2e77bdcbe891f7e0b9dd2ac01d56435f0b
-    git apply ../../spec-test-script/ignore_cases.patch
+
+    # aot multi-module is different with non-aot multi-module
+    if [[ ${ENABLE_MULTI_MODULE} == 1 &&  "${RUNNING_MODE}" == "aot" ]]; then
+      git apply ../../spec-test-script/ignore_cases_aot.patch
```

Comment:
How about renaming it to `ignore_cases_aot_multi_module.patch` to make it clear?

Reply:
✅ changed the file name as you recommended